canonical-web-and-design/ubuntu.com
# Create /ibm pages

## Done

- Created an /ibm page to move from partners.ubuntu.com

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ibm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare the text (mostly, some edits) to https://partners.ubuntu.com/ibm - I will create a copy doc.
- Test the contact us works


## Issue / Card

Fixes #7452

## Screenshots

![image](https://user-images.githubusercontent.com/441217/81839567-a0b64700-953f-11ea-8467-6db76b6ae76c.png)
